articles/stock-price-prediction-using-python/index.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/stock-price-prediction-using-python/
+title: Building a Stock Price Predictor Using Python
+description: In this tutorial, we build an AI neural network model in Python to predict stock prices.
+author: ahmad-mardeni
+date: 2020-12-30T00:00:00-12:00
+topics: [languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/stock-price-prediction-using-python/hero.png
+    alt: stock price prediction python
+---
+An artificial neural network is an interconnected group of nodes, inspired by the brain's simplification. Here, each circular node represents an artificial neuron, and an arrow represents a connection from the output of one artificial neuron to the input of another.

```

Comment:
@ahmadmardeni1 are you referencing an image here? If so, the image is missing. If not, I'm not sure what you're referring to when you say, "Here, each circular node represents... and an arrow represents..."

Reply:
Also, this is a strange way to begin the article. I'd suggest moving this paragraph down to the neural network section (i.e. under the 'What is RNN?' heading).

Reply:
@ninjaginja Hello, it is the Wikipedia definition for the artificial neural network. I will move it.

Reply:
In that case, you also need to reference it with a link. Any text that you copy from another source must be properly cited in order to adhere to plagiarism guideline. Please make sure to cite any additional sources/images that may be missing.

Reply:
@ninjaginja Hello, it is the Wikipedia definition for the artificial neural network. I don't think we can move it down because we are talking about recurrent neural network there. I will delete it.

Reply:
Instead of saying, According to Wikipedia, an RNN is defined as follows, mention the definition and insert the link. I pointed that out because Wikipedia didn't define RNN. Saying, according to Wikipedia, would be erroneous.